skyd/skyd.go

```diff
@@ -0,0 +1,16 @@
+package skyd
+
+type (
+	// Client allows us to call the local skyd instance.
+	Client struct {
+	}
+)
+
+/*
+TODO
```

Comment:
as with the database, I feel like a skyd client like this is something we keep repeating across our repositories, we should probably at one point merge all of those together, or even better tweak skyd's client so we can use it in external repositories such as this one

Reply:
While I do agree that we should move the DB stuff to a common library, other clients do not necessarily fall into the same bucket. Different services use different `skyd` endpoints and make different assumptions in order to be as easy to use as possible. As such, a client developed for one service might be inadequate for another.

Also, the base implementation of the `skyd` client is still the client within the `skyd` project itself. As you can see in #2, the local client implementation is tiny and all the heavy lifting is done by `gitlab.com/SkynetLabs/skyd/node/api/client`.